samples/experimental/custom_builder.yaml

```diff
@@ -6,13 +6,9 @@ metadata:
 spec:
   serviceAccount: default
   tag: sample/custom-builder
-  stack:
-    baseBuilderImage: cloudfoundry/cnb:bionic
+  stack: cflinuxfs3-stack
   store: sample-store
   order:
-  - group:
```

Comment:
Why drop the yarn group?

Reply:
we did this for a test. Should not have gotten checked in. Good catch.

Reply:
Well we dropped it because the sample store does not include it. We can either keep it dropped or add the yarn buildpackage to the sample store.

Reply:
I dont think we have a buildpackage for yarn just yet, so this would lead to a builder that cannot get built, unless we use another builder as a store

Reply:
Should we update the sample store to include cloudfoundry/bionic?

Reply:
I do like the idea of surfacing a builder as a store, but am worried it might confuse people who are browsing through it. But it does allow us to bring `yarn` back 🤷‍♂